### PR TITLE
Species likes/dislikes tweaks

### PR DESCRIPTION
**Pull request by MichaelCourtney**

### CODE REVIEW

File: default/scripting/species/SP_FULVER.focs.txt [hunk 1]
```diff
@@ -42,6 +42,15 @@ Species
         [[LARGE_PLANET]]
         [[NARROW_EP]]
         [[STANDARD_SHIP_SHIELDS]]
+		
+		EffectsGroup
```
Comment (geoffthemedio) [inline default/scripting/species/SP_FULVER.focs.txt]: Weird indenting. Are there tabs used?

Comment (MichaelCourtney) [reply default/scripting/species/SP_FULVER.focs.txt]: yeah looks like its 2 tabs, notepad++ just defaulted to that when I started a new line. Its in line with [[STANDARD_SHIP_SHIELDS]] when I look at it in npp

**Comment by MichaelCourtney** [comment]
Better?

File: default/scripting/species/SP_FULVER.focs.txt (hunk 1 above)
Comment (geoffthemedio) [reply default/scripting/species/SP_FULVER.focs.txt]: should replace tabs with 4 spaces

Comment (MichaelCourtney) [reply default/scripting/species/SP_FULVER.focs.txt]: should be fixed below

**Comment by geoffthemedio** [comment]
Since you're changing scripts that are visible to the checksums, could you also update the expected checksums for the tests:
````
5: /home/runner/work/freeorion/freeorion/test/system/TestChecksum.cpp:22: error: warning: in "TestChecksum/compare_checksum": FO_CHECKSUM_NAMED_VALUEREF expected 2351423 was 2465801
5: /home/runner/work/freeorion/freeorion/test/system/TestChecksum.cpp:22: error: warning: in "TestChecksum/compare_checksum": FO_CHECKSUM_TECH expected 5454078 was 5355498
````

**Comment by MichaelCourtney** [comment]
Is that correct now? In general how do I know when the expected checksums need adjusting and what the correct values are?

**Comment by geoffthemedio** [comment]
You can check the result of the tests on github for your pull request, or run the test locally. It's likely to change the checksums if you modify content scripts.

**Pull request merged**